apps/events-helsinki/src/constants.ts

```diff
@@ -51,3 +51,65 @@ export const FEEDBACK_LINKS = {
   en: 'https://www.hel.fi/helsinki/en/administration/participate/feedback',
   sv: 'https://www.hel.fi/helsinki/sv/stad-och-forvaltning/delta/feedback',
 };
+
+export const TARGET_GROUP_AGE_GROUPS_IN_ORDER = [
+  'babies',
+  'children',
+  'youth',
+  'adults',
+  'seniors',
+] as const;
+
+export type AgeGroup = (typeof TARGET_GROUP_AGE_GROUPS_IN_ORDER)[number];
+
+/**
+ * Maps age group names to their corresponding keyword IDs.
+ * The use of a simple `Record` provides type safety and a clear structure.
+ *
+ * @see https://api.hel.fi/linkedevents/v1/keyword/ to check keyword-related data.
+ * @see https://helsinkisolutionoffice.atlassian.net/browse/TH-1327 for spec.
+ */
+export const AGE_GROUP_KEYWORDS: Record<AgeGroup, readonly string[]> = {
+  babies: ['yso:p20513', 'yso:p15937'], // Vauvaperheet
```

Comment:
Small thing: could give description of both yso keywords:
```suggestion
  babies: [
    'yso:p20513', // Vauvaperheet
    'yso:p15937', // Vauvat
  ], 
```